specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+          "default": "CollectionReference",
+          "type": "string"
+        },
+        "referenceName": {
+          "description": "The name of reference",
+          "type": "string"
+        }
+      }
+    },
+    "DataFabricPolicyContentDetails": {
+      "type": "object",
+      "properties": {
+        "policyType": {
+          "type": "string"
+        }
+      }
+    },
+    "DataFabricPolicyContent": {
+      "type": "object",
+      "properties": {
+        "description": {
+          "type": "string"
+        },
+        "version": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "decisionRules": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DecisionRule"
+          }
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        },
+        "collection": {
+          "$ref": "#/definitions/CollectionReference"
+        },
+        "policyDetails": {
+          "$ref": "#/definitions/DataFabricPolicyContentDetails"
+        }
+      }
+    },
+    "DataPolicy": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DataFabricPolicyContent"
+        },
+        "name": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        }
+      }
+    },
+    "ErrorModel": {
+      "description": "The error model for policy",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target",
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "DataPolicyList": {
+      "required": [
+        "values"
+      ],
+      "type": "object",
+      "properties": {
+        "values": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        },
+        "count": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "DataPolicyRoleProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "The provisioningState of role",
```

Comment:
add space

Reply:
Isn't this an enumeration of known strings that can be listed?

Reply:
Added the three possible values for enumeration